models/postgrest/sql.go

```diff
@@ -0,0 +1,168 @@
+package postgrest
+
+import (
+	"fmt"
+	"scratchdata/util"
+	"strings"
+)
+
+const (
+	_ sqlOpertorFlag = 1<<iota - 1
+	// sqlOpNeedsParens represents operators whose operands should be wrapped in parentheses
+	sqlOpNeedsParens
+)
+
+var (
+	// sqlKeywords is a list of known SQL keywords that should not be treated as strings
+	sqlKeywords = map[string]bool{
+		"true":  true,
+		"false": true,
+		"null":  true,
+	}
+
+	// sqlOpertors is mapping of supported Postgrest operators to their SQL equivalent
+	// based on https://postgrest.org/en/stable/references/api/tables_views.html
+	sqlOpertors = map[string]sqlOpertor{
+		"eq":         {S: "="},
+		"gt":         {S: ">"},
+		"gte":        {S: ">="},
+		"lt":         {S: "<"},
+		"lte":        {S: "<="},
+		"neq":        {S: "<>"},
+		"like":       {S: "like"},
+		"ilike":      {S: "ilike"},
+		"match":      {S: "~"},
```

Comment:
I think we can just keep eq, gt, gte, lt, lte, neq, like, ilike, in, is, not, or, and, all, any. For `all` and `any`, let's translate those into the equivalent AND/OR sql clauses.